testsuite/features/init_clients/sle_ssh_minion.feature

```diff
@@ -58,23 +56,20 @@ Feature: Bootstrap a Salt host managed via salt-ssh
     Then I should see a "SUSE Linux Enterprise Server 15 SP2" text
     And vendor change should be enabled for "SP migration" on "ssh_spack_migrated_minion"
 
-@service_pack_migration
-  Scenario: Check the migration is successful for this SSH minion
+@proxy
+  Scenario: Check connection from SSH minion to proxy
     Given I am on the Systems overview page of this "ssh_spack_migrated_minion"
     When I follow "Details" in the content area
-    Then I should see a "SUSE Linux Enterprise Server 15 SP2" text
-    And vendor change should be enabled for SP migration on "ssh_spack_migrated_minion"
+    And I follow "Connection" in the content area
+    Then I should see "proxy" short hostname
 
-@service_pack_migration
-@ssh_minion
-  Scenario: Install the latest Salt on this SSH-managed minion
+  Scenario: Install the latest Salt on this minion
     When I enable repositories before installing Salt on this "ssh_spack_migrated_minion"
     And I install Salt packages from "ssh_spack_migrated_minion"
     And I disable repositories after installing Salt on this "ssh_spack_migrated_minion"
 
-@ssh_minion
   Scenario: Subscribe the SSH-managed SLES minion to a base channel
-    Given I am on the Systems overview page of this "ssh_minion"
+    Given I am on the Systems overview page of this "ssh_spack_migrated_minion"
```

Comment:
I think that at this point the migration was successful, so we might start calling it "ssh_minion" ?

Reply:
I think it's more readable, at least for me, if just at the moment that we migrate it, we start calling as ssh_spack_migrated_minion until the end of the feature. Otherwise, this swapping of names can get confusing.